src/fonduer/supervision/labeler.py

```diff
@@ -356,30 +361,38 @@ def _after_apply(
 
     def get_gold_labels(
         self, cand_lists: List[List[Candidate]], annotator: Optional[str] = None
```

Comment:
Can we add an option to support returning sparse/dense matrix here?

Reply:
No adding such an option here.
Label matrix shifter and unshifter have been implemented at 7fc08d985a65e08ff117d4bc66d372fbd30546e9.